client/js/components/procedure/admin/DpAddOrganisationList.vue

```diff
@@ -29,8 +103,8 @@
               v-if="selectedItems.length">
               {{ selectedItems.length }} {{ (selectedItems.length === 1 && Translator.trans('entry.selected')) || Translator.trans('entries.selected') }}
             </span>
-          </div><!--
-       --><div class="u-2-of-3 text-right inline-block space-inline-s">
+          </div>
+          <div class="u-2-of-3 text-right inline-block space-inline-s">
```

Comment:
the tailwind equivalent of `u-2-of-3` would be `w-2/3`. the tailwind equivalent of `space-inline-s` is `space-x-2` (i guess).

Reply:
right, done here https://github.com/demos-europe/demosplan-core/pull/2630/commits/cb457846ceee564184e8828a383186c9713faac1